src/execution/isolate.cc

```diff
@@ -1999,7 +2004,7 @@ Object Isolate::UnwindAndFindHandler() {
 
   // Special handling of termination exceptions, uncatchable by JavaScript and
   // Wasm code, we unwind the handlers until the top ENTRY handler is found.
-  bool catchable_by_js = is_catchable_by_javascript(exception);
+  bool catchable_by_js = !!exception && is_catchable_by_javascript(exception);
```

Comment:
I don't think this change is necessary?  Undefined is an object like any other, scripts can `throw undefined` too of course.

Reply:
It was a build error, too